### PR TITLE
EZP-28968: Error when creating URL alias at root level

### PR DESCRIPTION
**Pull request by ViniTou**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28968?
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Disable url alias creation under root path for root locations.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/lib/Form/EventListener/BuildPathFromRootListener.php [hunk 1]
```diff
@@ -41,6 +41,9 @@ public function onPreSubmitData(FormEvent $event)
         $data = $event->getData();
         if (!array_key_exists('site_root', $data) || false === (bool)$data['site_root']) {
             $location = $this->locationService->loadLocation($data['location']);
+            if ($location->depth <= 1) {
```
Comment (mikadamczyk) [inline src/lib/Form/EventListener/BuildPathFromRootListener.php]: We mostly use Yoda condition

File: src/lib/Form/Type/Content/CustomUrl/CustomUrlAddType.php [hunk 2]
```diff
@@ -106,5 +107,29 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+        $builder->addEventListener(FormEvents::PRE_SET_DATA, [
+            $this,
+            'disableSiteRootCheckboxIfRootLocation',
+        ]);
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormEvent $event
+     */
+    public function disableSiteRootCheckboxIfRootLocation(FormEvent $event)
```
Comment (mikadamczyk) [inline src/lib/Form/Type/Content/CustomUrl/CustomUrlAddType.php]: Another events are in separate files

File: src/lib/Form/EventListener/BuildPathFromRootListener.php (hunk 1 above)
Comment (alongosz) [reply src/lib/Form/EventListener/BuildPathFromRootListener.php]: > We mostly use Yoda condition

But that's optional, we don't force it. Here what we have looks more readable to me than Yoda :P

Comment (mikadamczyk) [reply src/lib/Form/EventListener/BuildPathFromRootListener.php]: Of course, it is only social contract ;)

**Pull request merged**